src/components/api-requests/everything.tsx

```
@@ -199,33 +206,70 @@ export default function Everything() {
+                      value={String(selectedExample)}
+                      onValueChange={(value) =>
+                        setSelectedExample(Number(value))
+                      }
+                    >
+                      <SelectTrigger className="w-[200px] font-mono text-xs">
+                        <SelectValue />
+                      </SelectTrigger>
+                      <SelectContent>
+                        {selectedResource.examples.map((example, idx) => (
+                          <SelectItem
+                            key={`example-${example.resourceType}-${idx}`}
```

Biome (2.4.16)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)